### ISSUE TITLE
🍫 scroll containers in devtools should show horizontal scrollbars

### ISSUE THREAD

**Issue by bartveneman** [projectwallace/website#17]
The scroll container nesting should be fixed so that horizontal scrollbars are always shown and not only when scrolling down into the devtools panels.

Apparently a hard problem to solve. https://bsky.app/profile/veneman.dev/post/3lwfeyeknrk2x
